Source/MediaInfo/Audio/File_Adm.cpp

```diff
@@ -2398,10 +2398,11 @@ class file_adm_private : public tfsxml
     file_adm_private()
     {
         auto OldLocale_Temp = setlocale(LC_NUMERIC, nullptr);
-        if (*OldLocale_Temp != 'C' || *(OldLocale_Temp + 1)) {
-            OldLocale = OldLocale_Temp;
-            setlocale(LC_NUMERIC, "C");
-        }
+        if (OldLocale_Temp)
+            if (*OldLocale_Temp != 'C' || *(OldLocale_Temp + 1)) {

```

Comment:
Better to avoid an extra indetation.
if (OldLocale_Temp && (*OldLocale_Temp != 'C' || *(OldLocale_Temp + 1)))